mvlearn/cluster/mv_spectral.py

```diff
@@ -161,60 +161,13 @@ class MultiviewSpectralClustering(BaseEstimator):
     0.872
 
     '''
-    def __init__(self, n_clusters=2, n_views=2, random_state=None,
+    def __init__(self, n_clusters=2, random_state=None,
```

Comment:
Change the docstring Parameters section to reflect this change (no more n_views)

Reply:
Will do